README.md

```diff
@@ -128,6 +128,48 @@ charge.lastResponse.requestId // see: https://stripe.com/docs/api/node#request_i
 charge.lastResponse.statusCode
 ```
 
+### `request` and `response` events
+
+The Stripe object emits `request` and `response` events.  You can use them like this:
+
+```js
+var stripe = require('stripe')('sk_test_...');
+
+function onRequest(request) {
+  // Do something.
+}
+
+stripe.on('request', onRequest);
+
+stripe.removeListener('request', onRequest);
+// OR: stripe.off('request', onRequest);
```

Comment:
Again a bit of a nit, but just to minimize the new API we expose and so that we can provide a strong recommendation, can we just pick one of the two here? :) We're not `EventEmitter` compatible anyway, so maybe `off` would be fine (it's fairly unlikely people will need to use this remove function much anyway).

Just remember that we can always add a new alias if we find a great reason for it, but we can't easily remove one.